openshift_pipelines/uninstalling-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="uninstalling-pipelines"]
+= Uninstalling OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: uninstalling-pipelines
+
+toc::[]
+
+Uninstalling the {pipelines-title} Operator is a two step process. First, you delete the custom resources, which are added by default when you install the Operator. Second, you uninstall the {pipelines-title} Operator.
```

Comment:
IMO, it would make more sense to write, "First, you delete the custom resources that were added by default when you installed the Operator." You might also consider breaking these into bullets for scannability:

> Uninstalling the OpenShift Pipelines Operator is a two-step process: 
> 
> - Delete the custom resources that were added by default when you installed the Operator.
> - Uninstall the OpenShift Pipelines Operator.

Reply:
I liked this breakdown approach. Instead of bullets, I have used numbered list, since we say two-step process.